napari/_qt/qt_viewer.py

```diff
@@ -202,8 +227,24 @@ def _update_palette(self, palette):
 
     def on_mouse_move(self, event):
         """Called whenever mouse moves over canvas.
+        when axis is not None and the camera is turntable, on moving the mouse
+        one can rotate around the vertical axes.
         """
         layer = self.viewer.active_layer
+        if (
```

Comment:
should i revert these changes too @sofroniewn

Reply:
these changes are needed for the camera to work properly, right(?), and should stay - you should just revert the file IO changes